src/content/apis/social/shrtcode.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Shrtcode API
+description: Free API to shorten Links
+categories:
+  - social
+features:
+  https: true
+  official: true
```

Comment:
Official is actually meant for APIs of large companies and such. Better to change it for now
```suggestion
  official: false
```